src/file-uploader/file.component.ts

```diff
@@ -43,7 +43,7 @@ import { FileItem } from "./file-item.interface";
 		</span>
 	`
 })
-export class File {
+export class FileComponent {
```

Comment:
Can we add an export at the end of the file to do a `export FileComponent as File` as well? Just in case anyone was importing the file component directly, we don't want to break them